### ISSUE TITLE
Getting Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '_c') (Vuejs 3)

### ISSUE THREAD

**Issue by hazem-taha** [LouisMazel/vue-phone-number-input#159]
Hello,

My app is (Inertia + Vuejs 3 composition API)

```
import VuePhoneNumberInput from "vue-phone-number-input";`

export default defineComponent({
  components: {
    VuePhoneNumberInput,
  },
...
...
```

`<VuePhoneNumberInput v-model="form.phone" />`

Getting the below error when trying to load the page containing the phone element..

![image](https://user-images.githubusercontent.com/46519001/135464126-be2dd3ed-204b-44ca-8f74-b731383a052f.png)

**Comment by LouisMazel** [comment]
Hi @hazem-taha,

This plugin works only with Vue2

I will release soon the Vue3 version on maz-ui

Stay tuned

**Comment by PMLS3** [comment]
Hi @LouisMazel , thanks for a great library! Do you perhaps have a timeframe on when you will release for vue3?

**Comment by hathemi** [comment]
this bug still exist ! any solution ?

**Comment by LouisMazel** [comment]
Hi,

I'm currently working on the vue 3 version, is still in beta for the moment : 

**Getting Started**: https://louismazel.github.io/maz-ui-3/guide/getting-started.html
**MazPhoneNumberInput**: https://louismazel.github.io/maz-ui-3/components/maz-phone-number-input.html

Be careful, so many things can change in the futur

**Comment by LouisMazel** [comment]
Let me know if you found any bugs 😊

**Comment by hathemi** [comment]
https://louismazel.github.io/maz-ui-3/components/maz-phone-number-input.html#feature-list 
i have tried this component and here's what i've got
`
MazPhoneNumberInput.js?93d8:15 GET http://localhost:8100/chevron-down.svg 404 (Not Found)`

also it **Doesn't show any flags** 
did I do something wrong?

**Comment by LouisMazel** [comment]
@hathemi 
Fixed in version 3.0.0-beta.20
